### PR TITLE
Add mailsend and Loader app source

### PR DESCRIPTION
**Pull request by kbembedded**

### CODE REVIEW

**Review by CaptainNic — CHANGES_REQUESTED** [review]
Did you mean to commit the `mailsend.exe`?

File: src/tools/mailsend/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+The `Loader` app is a Mailstation OS app. It always runs from `0x4000` no matter which dataflash page the app is located on. It sets RAM page 1 to slot8000, loads in the data to there, and then jumps to `0x8000`. Because of this, any apps loaded with this method need to be compiled to run with an origin of `0x8000`.
+
+
+## Mailsend
+The `mailsend` tool is the PC side to send data directly to the `Loader` app.
+
+Currently, the sources have not been compiled in a very long time. In this folder is a Windows executable, what is probably the source code to it, and a Windows DLL (it is a 64-bit DLL) for parallel port communication. The executable has been tested and confirmed on a 64-bit Windows 7 machine.
+
+
+## TODOs
```
Comment (CaptainNic) [inline src/tools/mailsend/README.md]: Should add these as issues.

**Review by CaptainNic — APPROVED** [review]
nevermind, RTFM. Looks good :+1:

File: src/tools/mailsend/README.md (hunk 1 above)
Comment (kbembedded) [reply src/tools/mailsend/README.md]: Yeah, probably, will do that.

Comment (kbembedded) [reply src/tools/mailsend/README.md]: Done.

**Review by CaptainNic — APPROVED** [review]

**Pull request merged**